src/Operation/BulkWrite.php

```diff
@@ -325,6 +326,8 @@ public function execute(Server $server)
             $type = key($operation);
             $args = current($operation);
 
+            assert(is_array($args));
```

Comment:
Is this necessary? Since `$operations` is private couldn't you also just add an `array` property type hint to it?

Reply:
I believe the error crept up due to a change in the baseline, so I've removed this `assert` call and updated the baseline instead. Fixing this would've required updating types in more than just this file, which is why it was initially deferred. We have follow-up tickets to fix issues like this one.